molecule/yaml_config_validation/converge.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Converge
+  hosts: all
+  vars_files:
+    - vars.yml
+  collections:
+    - middleware_automation.wildfly
+  roles:
+    - wildfly_install
+    - wildfly_systemd
+    - wildfly_driver
+    - wildfly_utils
+  tasks:
+    - name: Copying the setup file
+      ansible.builtin.copy:
+        src: wildfly_yaml_configuration.yml
+        dest: "{{ wildfly_home }}/standalone/wildfly_yaml_configuration.yml"
+        mode: 0644
+
+    - name: Set yaml conf file destination
+      ansible.builtin.set_fact:
+        yaml_conf_file: "{{ wildfly_yaml_config_location }}/{{ yaml_service }}{{ wildfly_yaml_config_file_suffix }}"
+      when:
+        - not yaml_conf_file is defined
+
+    - name: "Deploy yaml configuration for : {{ yaml_conf_file }}"
+      ansible.builtin.template:
+        src: "{{ wildfly_config_file_template }}"
+        dest: "{{ yaml_conf_file }}"
+        group: root
+        owner: root
+        mode: 0640
+      register: daemon_reload
+      become: yes
+      tags: molecule-idempotence-notest
+
+    - name: "Perform daemon-reload to ensure the changes are picked up"
```

Comment:
If you provided the config BEFORE the playbook is fired up, you should not need to reload the service (AFIAU)

Reply:
Done, Thanks